docs/aerodromes/classc/Williamtown.md

```diff
@@ -21,6 +21,63 @@ WLM ADC owns the airspace within the WLM MIL CTR A (`SFC`-`A050`). This may be a
   <figcaption>WLM ADC Airspace</figcaption>
 </figure>
 
+### SID Selection
+Aircraft planned via **KEXAR**, **LIMLO**, **MONDO**, **UBSON**, shall be assigned the **Procedural SID** that terminates at the appropriate waypoint.
+Aircraft who are not planned via those points or who are negative RNAV may be assigned a RADAR SID or a visual departure. 
+
+#### Coded Departures (Fast Jets)
+Visual Departures are commonly requested by high performance military jets and often processed in the form of a coded departure (e.g. CLASSIC1 departure), which can be found in the AD2 Sups Page for YWLM. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east and west of Williamtown, avoiding civilian traffic inbound to Sydney. 
+
+| Tracking Gate | Departure | Initial Constraint | Route |
+| ----- | ---------| --------------- | ------------------------ |
+| G1-G2 | PEPPER 1 | `A080` at ERVEV | ERVEV G1/G2 (Transition) 
+| G3-G7 | CLASSIC 1| `F125` at G3-G7 |
+| G9    | RADAR    | 'F125' at G9    |
+| UKDIP | CLASSIC 1| 'A090' at UKDIP |
+| OLVEP | CLASSIC 1| 'F130' at OLVEP |
+
+!!! note
+    ACFT Departing RWY 12 for G3-7 are required to avoid SAWR on the Classic 1.
+    ACFT Departing on Classic 1 will be issued a direction of turn during take off clearance, preferablly to the south in order not to impede on the SAWR boundries on the departure.
+
+!!! example
+    WGTL21 plans to enter M550 via Gate 4 for Military Training.
+    **WLM ADC**: "WGTL21, cleared to YWLM via Gate 4, flight planned route. Runway 12, Classic 1 Departure. Climb to `F125`. Squawk 5001, Departure Frequency 133.300"  
+
+!!! example
+    MAPL03 was assiged the CLASSIC 1 departure with their initial clearance. 
+    **WLM ADC**: "MAPL03, make left turn, reach `F125` by Gate 3 , cleared for takeoff"  
+    **MAPL03**: "Make left turn, reach `F125` by Gate 3, cleared for takeoff, MAPL03"  
+
+!!! note
+    Since the Coded Departures are not SIDs, WLM ACD needs to write the name of the coded departure in the Global Ops Field of the FDR and amend the flight plan route to include all relevant tracking points.
+    The most often departure to the M550 area is via Gate 4.
+
+### Gates for the SUA
+There are 8 gates to facilitate departures and arrivals into these Training Airspace, based on the WLM Tacan. 
+WLM Eastern Airspace
+
+| Gate Name | Ident | Bearing (°M) | Distance (nm) | Restricted Area |
+| Gate 1 | G1 | 029 | 25 | R586 |
+| Gate 2 | G2 | 050 | 25 | R586 | 
+| Gate 3 | G3 | 068 | 25 | M550 |
+| Gate 4 | G4 | 088 | 25 | M550 |
+| Gate 5 | G5 | 105 | 25 | M550 |
+| Gate 6 | G6 | 125 | 25 | M550 |
+| Gate 7 | G7 | 135 | 25 | M550 |
+| Gate 9 | G9 | 007 | 25 | R585 |
+| UKDIP | UKDIP |311 | 25 | R560 |
+| OLVEP | OLVEP |033 | 25 | R596 |
+
+<figure markdown>
+![WLM Gates](){ width="700" }
+  <figcaption>WLM Gates</figcaption>
+</figure>
```

Comment:
Just seen this, looks like this `<figure>` doesn't reference an image

Reply:
Havent finished it yet, just a reminder that I need to put a diagram in

Reply:
@mattkelly4 Anychance you can make the diagram in vatsys? I dont know how to get the restriced airspace to show

Reply:
@ST6-Bravo I won't have a chance for a while unfortunately but if you want to give it a crack, use the Restricted Areas window to find the PRD, then set the activation time from `0000` to `2400` and hit apply. It'll appear on the map
![image](https://github.com/user-attachments/assets/326309d8-35e0-4590-b615-503c0c87f91c)

Reply:
Will give it a crack then

Reply:
Just dont know how to get the fancy text like in the tindal one to appear, guessing @alphadelta332 would know?

Reply:
> Just dont know how to get the fancy text like in the tindal one to appear, guessing @alphadelta332 would know?

That was just done up in Photoshop. Do you have the tools for that? It doesn't matter if the formatting isn't identical, as long as the diagram is clear

Reply:
I dont have photoshop, so can you make it?

Reply:
When I get a chance i'll have a go at making it